.github/workflows/build-pr.yml

```diff
@@ -39,21 +39,12 @@ jobs:
     name: Test Hazelcast Membership
     runs-on: ubuntu-latest
     outputs:
-      is-hazelcast: ${{ steps.test-membership.outputs.is-member }}
+      is-hazelcast: ${{ steps.test-membership.outputs.check-result }}
 
     steps:
-      # checkout the hazelcast/hazelcast-cpp-client repository
-      # bare minimum - just to use actions
-      - name: Checkout code
```

Comment:
why deleted `Checkout code` ?

Reply:
> why deleted `Checkout code` ?

Only for the `test-membership` step - it does a `checkout` to get a copy of the `test-membership` action.

By referencing the (remote) `hazelcast/hazelcast-tpm/membership` action, that's done implicitly.